homeassistant/components/rointe/sensor_descriptions.py

```diff
@@ -0,0 +1,74 @@
+"""Sensor descriptions for Rointe."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import ENERGY_KILO_WATT_HOUR, POWER_WATT, TEMP_CELSIUS
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.typing import StateType
+
+from .device_manager import RointeDevice
+
+
+@dataclass
+class RointeSensorEntityDescriptionMixin:
+    """Define a description mixin for Rointe sensor entities."""
+
+    value_fn: Callable[[RointeDevice], StateType]
+    last_reset_fn: Callable[[RointeDevice], datetime | None]
+
+
+@dataclass
+class RointeSensorEntityDescription(
+    SensorEntityDescription, RointeSensorEntityDescriptionMixin
+):
+    """Define an object to describe Rointe sensor entities."""
+
+
+SENSOR_DESCRIPTIONS = [
```

Comment:
It's common practice to put these descriptions in the same file containing entity class definitions (in this case, `sensor.py`).